front/app/containers/ProjectsShowPage/shared/pb/PBExpenses.tsx

```diff
@@ -407,10 +413,31 @@ class PBExpenses extends PureComponent<Props & InjectedIntlProps, State> {
             </ProgressBar>
 
             <Footer viewMode={viewMode}>
+              {viewMode === 'column' && (
+                <TotalBudgetColumn aria-hidden>
+                  <BudgetLabel>
+                    <FormattedMessage
+                      {...(minBudget === maxBudget
```

Comment:
If min = max budget, we don't show min required budget nor 'your budget', just the fixed selection they need to submit.